app/service/rest_svc.py

```diff
@@ -302,15 +302,36 @@ async def _construct_adversary_for_op(self, adversary_id):
         return Adversary.load(dict(adversary_id='ad-hoc', name='ad-hoc', description='an empty adversary profile', atomic_ordering=[]))
 
     async def _update_global_props(self, sleep_min, sleep_max, watchdog, untrusted, implant_name, bootstrap_abilities):
-        if implant_name:
-            self.set_config(name='agents', prop='implant_name', value=implant_name)
-        if bootstrap_abilities:
-            abilities = self.get_config(name='agents', prop='bootstrap_abilities')
-            abilities.append(bootstrap_abilities)
+        """Update global agent properties
+
+        :param sleep_min: Beacon min sleep time (seconds)
+        :type sleep_min: int
+        :param sleep_max: Beacon max sleep time (seconds)
+        :type sleep_max: int
+        :param watchdog: Watchdog timer (seconds)
+        :type watchdog: int
+        :param untrusted: Untrusted timer (seconds)
+        :type untrusted: int
+        :param implant_name: Agent file name
+        :type implant_name: str
+        :param bootstrap_abilities: Comma-separated ability UUIDs
+        :type bootstrap_abilities: str
+        """
         self.set_config(name='agents', prop='sleep_min', value=sleep_min)
         self.set_config(name='agents', prop='sleep_max', value=sleep_max)
         self.set_config(name='agents', prop='untrusted_timer', value=untrusted)
         self.set_config(name='agents', prop='watchdog', value=watchdog)
+        if implant_name:
+            self.set_config(name='agents', prop='implant_name', value=implant_name)
+        if bootstrap_abilities:
+            abilities = []
+            ability_ids = [ability_id.strip() for ability_id in bootstrap_abilities.split(',') if ability_id.strip()]
```

Comment:
If a user enters an invalid UUID, they are not notified that it is invalid. The UI only indicates that the agent configuration has been updated. You would have to look at the console to know that it failed.

Reply:
Added stream message to display ignored bootstrap abilities in https://github.com/mitre/caldera/pull/1846/commits/d699ec88226f1f8a0f0e5c2d2d991e723f1e941c